### PR TITLE
Add Benben Delete

### PR DESCRIPTION
**Pull request by Lawsonwang**
-M benben-delete

### CODE REVIEW

**Comment by wxh06** [comment]
Thanks for your contribution!
However, it is suggested to run [ESLint](https://eslint.org/) locally and fix them (e.g. via the flag `--fix`) if there are any problems with code style before committing.

File: src/modules/benben.js [hunk 1]
```diff
@@ -112,4 +113,63 @@ mod.reg("benben", "全网犇犇", "@/", {
 mod.reg("benben-quickpost", "CtrlEnter发送犇犇", "@/", null, () =>
     $("textarea").whenKey("CtrlEnter", () => $("#feed-submit").click())
 , `
-`, "module")
\ No newline at end of file
+`, "module")
+
+mod.reg("benben-delete", "一键删犇", "@/", null, () => {
+    var delete_ben = function() {
```
Comment (YuzhenQin) [inline src/modules/benben.js:119]: `const delete_ben = function() {`

File: src/modules/benben.js [hunk 2]
```diff
@@ -112,4 +113,63 @@ mod.reg("benben", "全网犇犇", "@/", {
...
+        var l=$("#feed>li>div.am-comment-main>header>div>a:nth-child(2)")
```
Comment (YuzhenQin) [inline src/modules/benben.js:120]: `const l=$("#feed>li>div.am-comment-main>header>div>a:nth-child(2)")`

File: src/modules/benben.js [hunk 3]
```diff
@@ -112,4 +113,63 @@ mod.reg("benben", "全网犇犇", "@/", {
...
+                            setTimeout(function(){ location.reload() }, 3000)
+                        }
+                    }
+                }
+            )
+        }
+        f(0)
+    }
+
+    var find_ben = function() { // Note: Loading bens
```
Comment (YuzhenQin) [inline src/modules/benben.js:141]: `const find_ben = function() { // Note: Loading bens`

File: src/modules/benben.js [hunk 4]
```diff
@@ -112,4 +113,63 @@ mod.reg("benben", "全网犇犇", "@/", {
...
+        function load(){
+            console.log("page "+feedPage)
+            $.get("/feed/"+feedMode+"?page="+feedPage, function(resp){
+                $feed.append(resp)
```
Comment (YuzhenQin) [inline src/modules/benben.js:145]: `uindow.$feed.append(resp)`

**Pull request closed without merging**